Make hop's link rules honor flags set by the --blflags configure argument.

--- hop-2.4.0/src/Makefile	2015-05-05 19:41:04.800151036 -0500
+++ hop-2.4.0/src/Makefile	2015-05-05 19:40:40.916150417 -0500
@@ -69,10 +69,10 @@
 	$(MAKE) link.$(LINK) DEST=$@
 
 link.dynamic:
-	@ $(call link,$(BIGLOO),$(BCFLAGS),$(BCFLAGSDEV),$(OBJECTS),-o,$(DEST))
+	@ $(call link,$(BIGLOO),$(BCFLAGS) $(BLFLAGS),$(BCFLAGSDEV),$(OBJECTS),-o,$(DEST))
 
 link.static:
-	@ $(call link,$(BIGLOO),$(BCFLAGS),$(BCFLAGSDEV),-static-all-bigloo $(OBJECTS),-o,$(DEST))
+	@ $(call link,$(BIGLOO),$(BCFLAGS) $(BLFLAGS),$(BCFLAGSDEV),-static-all-bigloo $(OBJECTS),-o,$(DEST))
 
 link.library:
 	echo "***ERROR: link.library not currently supported!"
--- hop-2.4.0/hopc/Makefile	2013-01-30 07:17:59.000000000 -0600
+++ hop-2.4.0/hopc/Makefile	2015-05-05 19:45:21.876157699 -0500
@@ -62,7 +62,7 @@
 	mkdir -p $@
 
 $(BUILDBINDIR)/$(EXEC): .afile .etags $(OBJECTS)
-	@ $(call link,$(BIGLOO),$(BCFLAGS),,$(OBJECTS),-o,$@)
+	@ $(call link,$(BIGLOO),$(BCFLAGS) $(BLFLAGS),$(BCFLAGSDEV),$(OBJECTS),-o,$@)
 
 $(BUILDBINDIR)/$(EXEC).jar: .afile .etags .jfile $(BGL_CLASSES) META-INF/MANIFEST.MF jvm-stdlibs jvm-share jvm-lib
 	$(JAR) $@ META-INF/MANIFEST.MF -C o/class_s .
--- hop-2.4.0/hophz/Makefile	2013-01-30 07:17:59.000000000 -0600
+++ hop-2.4.0/hophz/Makefile	2015-05-05 19:59:42.996180030 -0500
@@ -16,9 +16,6 @@
 -include ../etc/Makefile.hopconfig
 -include ../etc/Makefile.version
 
-BLFLAGS		= 
-BLINKFLAGS 	= -suffix hop
-
 #*---------------------------------------------------------------------*/
 #*    Target and Project                                               */
 #*---------------------------------------------------------------------*/
@@ -72,7 +69,7 @@
 	mkdir -p $@
 
 $(BUILDBINDIR)/$(EXEC): .afile .etags $(OBJECTS)
-	@ $(call link,$(BIGLOO),$(BCFLAGS),$(BLINKFLAGS),$(OBJECTS),-o,$@)
+	@ $(call link,$(BIGLOO),$(BCFLAGS) $(BLFLAGS),$(BCFLAGSDEV),$(OBJECTS),-o,$@)
 
 $(BUILDBINDIR)/$(EXEC).jar: .afile .etags .jfile $(BGL_CLASSES) META-INF/MANIFEST.MF jvm-stdlibs jvm-share jvm-lib
 	@ $(JAR) $@ META-INF/MANIFEST.MF -C o/class_s .
--- hop-2.4.0/hopsh/Makefile	2013-01-30 07:17:59.000000000 -0600
+++ hop-2.4.0/hopsh/Makefile	2015-05-05 19:46:36.060159626 -0500
@@ -60,7 +60,7 @@
 	mkdir -p $@
 
 $(BUILDBINDIR)/$(EXEC): .afile .etags $(OBJECTS)
-	@ $(call link,$(BIGLOO),$(BCFLAGS),$(BCFLAGSDEV),$(OBJECTS),-o,$@)
+	@ $(call link,$(BIGLOO),$(BCFLAGS) $(BLFLAGS),$(BCFLAGSDEV),$(OBJECTS),-o,$@)
 
 $(BUILDBINDIR)/$(EXEC).jar: .afile .etags .jfile $(BGL_CLASSES) META-INF/MANIFEST.MF jvm-stdlibs jvm-share jvm-lib
 	@ $(JAR) $@ META-INF/MANIFEST.MF -C o/class_s .
